src/0x.ts

```diff
@@ -194,11 +194,20 @@ export class ZeroEx {
             gasPrice,
         };
         this._web3Wrapper = new Web3Wrapper(provider, defaults);
-        this.token = new TokenWrapper(this._web3Wrapper);
```

Comment:
At the top of this method we need to asset that `config` conforms to the ZeroExConfig schema.